package.json

```diff
@@ -2,25 +2,33 @@
   "name": "eslint-visitor-keys",
   "version": "2.1.0",
   "description": "Constants and utilities about visitor keys to traverse AST.",
+  "type": "module",
   "main": "lib/index.js",
+  "exports": {
+    "import": "./lib/index.js",
+    "require": "./dist/eslint-visitor-keys.cjs"
+  },
   "files": [
     "lib"
   ],
   "engines": {
-    "node": ">=10"
+    "node": ">=12"
```

Comment:
Sure. To make it easier to merge, I've added a commit to mirror your current engines changes but can of course update later according to any subsequent changes.

Reply:
By the way, I've added subsequent commits with the expected commit messages, e.g., "Fix:", etc. so as to avoid wiping out the commit history. Not sure if you prefer my amending instead, though in any case, I can of course squash it later as deisred.

Reply:
no worries, we will squash the commits when merging. 😄